### PR TITLE
[zotac.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by J0WI** [comment]
Could you please complete the ruleset with the following subdomains?

Unsupported:
downloads.zotac.com
china.zotac.com
webuat.zotac.com
hkftp.zotac.com
old.zotac.com
dlasia.zotac.com
hkftp.zotac.com

WFM:
webdev.zotac.com

**Comment by youdly** [comment]
Added comment and ruleset for functional subdomains.

**Comment by J0WI** [comment]
Thanks for the completion. The rule lgtm so far, but our test are currently broken.

> Btw, usually I catch subdomains either by browsing around or using wolframalpha, any other suggestion?

See https://github.com/EFForg/https-everywhere/pull/3517#issuecomment-165904237

File: src/chrome/content/rules/ZOTAC.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		- hkftp				²
+		- old				²
+		- webuat			²
+
+	¹ refused
+	² timeout
+-->
+<ruleset name="ZOTAC">
+	<target host="zotac.com" />
+	<target host="china.zotac.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/ZOTAC.xml]: Mixed content

**Comment by J0WI** [comment]
Sorry, I forgot this one. Please exclude `china.zotac.com` and add a comment about the mixed content issues.

**Comment by J0WI** [comment]
@youdly are you still working on this?

**Comment by youdly** [comment]
Removed `china` subdomain.

**Pull request merged**